INSTALL.rst

```diff
@@ -175,14 +175,12 @@ To suggest a change in these instructions,
 Installing scikit-image for contributors
 ========================================
 
-We are assuming that you have a default Python environment already configured on
-your computer and that you intend to install ``scikit-image`` inside of it.
-
+We are assuming that you have already `installed Python <https://www.python.org/downloads/>`_
+on your computer and that you intend to use it for ``scikit-image``.
 We also make a few more assumptions about your system:
 
-- You have a C compiler set up.
-- You have a C++ compiler set up.
-- You are running a version of Python compatible with our system as listed
+- You have a C and C++ compiler set up.
```

Comment:
This line in particular is much easier with conda-forge (in my opinion) when trying to ensure a consistent set of compilers and feature are used across different versions of linux.

Reply:
What's the Windows side of this? I would have thought that Linux isn't much of an issue since most of the "help me compile this" issues in my memory are from Windows users.